src/System.Globalization/tests/Invariant/InvariantMode.cs

```diff
@@ -0,0 +1,796 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class InvariantModeTests
+    {
+        public static IEnumerable<object[]> Cultures_TestData()
+        {
+            yield return new object[] { "en-US" };
+            yield return new object[] { "ja-JP" };
+            yield return new object[] { "fr-FR" };
+            yield return new object[] { "tr-TR" };
+            yield return new object[] { "" };
+        }
+
+        private static string[] s_cultureNames = new string[] { "en-US", "ja-JP", "fr-FR", "tr-TR", "" };
+
+        public static IEnumerable<object[]> IndexOf_TestData()
+        {
```

Comment:
There is many differences compared to the data exist in other tests. this is the whole point for test invariant mode to ensure such differences.